guides/common/modules/ref_accessing-ansible-modules-docs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="accessing-ansible-modules-docs_{context}"]
+== Accessing {Project} Ansible Modules Documentation
+
+You can read the content collection documentation offline on the command line using the following command:
+
+----
+# ansible-doc redhat.satellite.[module_name]
+----
+
+[NOTE]
+====
+* At the time of writing, online documentation is only available on the Automation Hub at https://cloud.redhat.com/ansible/automation-hub/redhat/satellite/docs.
+* For more information about {Project} Ansible modules and example workflows, see https://www.ansible.com/blog/automating-red-hat-satellite-with-ansible.
```

Comment:
Let's not link this Blog post from docs.

Reply:
Done